testgrid/config.yaml

```diff
@@ -6625,6 +6634,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-regional-slow
     base_options: include-filter-by-regex=Volume%7Cstorage
     description: storage gke regional slow e2e tests for master branch
+  dashboard_names:
+  - sig-storage-local-static-provisioner
```

Comment:
I followed `sig-gcp-compute-persistent-disk-csi-driver` to add CI jobs in sub-dashboard under sig-storage.

https://github.com/kubernetes/test-infra/blob/f5ac6c3a74a392c00969aabaa458148b27ad9673/testgrid/config.yaml#L7739-L7744

https://github.com/kubernetes/test-infra/blob/f5ac6c3a74a392c00969aabaa458148b27ad9673/testgrid/config.yaml#L5934-L5938

Reply:
It looks like this is showing up as a new top-level dashboard instead of underneath sig-storage.  Maybe this is because we already have dashboard_tabs under the top-level sig-storage dashboard and we need to move all of those under a sub dashboard?

Reply:
Yup, that use of dashboard_names doesn't work. sig-storage is a dashboard right now; if you want it as a dashboard group, you'll need to modify it in line with the other dashboard groups (like https://github.com/kubernetes/test-infra/blob/f5ac6c3a74a392c00969aabaa458148b27ad9673/testgrid/config.yaml#L7589), which probably means renaming the current sig-storage dashboard as well.

Reply:
is it possible to create a sig-storage dashboard group, and put current sig-storage dashboard under it? like this:

- sig-storage (dashboard group)
  - sig-storage (dashboard)
  - sig-storage-local-static-provisioner (dashboard)

Reply:
We could call the dashboard sig-storage-kubernetes

Reply:
@michelle192837 is there a way to preview locally?

Reply:
Yeah, can't duplicate dashboard and dashboard group names, but naming the group sig-storage and the dashboard sig-storage-kubernetes should work.

Unfortunately, no way to preview locally :( But the changes should show up within 30 minutes after merge, if not sooner.

Reply:
Thanks!

Reply:
Np! Let me know if you're still hitting issues; I'm happy to review the next PR.

Reply:
fix https://github.com/kubernetes/test-infra/pull/10696